iambic/plugins/v0_1_0/github/github.py

```diff
@@ -16,6 +16,7 @@
 from urllib.parse import urlparse
 
 import github
+from cryptography.hazmat.primitives import serialization
```

Comment:
nit: should the json import above this file be `from iambic.core import noq_json as json` instead?